rllib/examples/envs/classes/multi_agent/footsies/test/footsies_suppress_unity_logs.py

```diff
@@ -0,0 +1,135 @@
+import os
+import platform
+import tempfile
+import time
+import unittest
+from contextlib import contextmanager
+from pathlib import Path
+
+from ray.rllib.env import EnvContext
+from ray.rllib.examples.envs.classes.multi_agent.footsies.footsies_env import (
+    env_creator,
+)
+
+# Detect platform and choose appropriate binary
+if platform.system() == "Darwin":
+    binary_to_download = "mac_headless"
+elif platform.system() == "Linux":
+    binary_to_download = "linux_server"
```

Comment:
This change is great, but now user cannot configure if they want headless or windowed application.

We can change it so that OS is detected automatically (like you did!) and just let users select either headless or windowed variant (default is headless).

WDYT?